applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -19,6 +19,7 @@ The *Developer* perspective in the web console provides you the following option
 ** *Database*: Select the required database service and add it to your application.
 ** *Operator Backed*: Select and deploy the required Operator-managed service.
 ** *Helm chart*: Select the required Helm chart to simplify deployment of applications and services.
+** *Devfile*: Select a devfile from the Devfile registry to declaratively define a development environment.
```

Comment:
If Devfile registry is a gui element, make it bold. Also, as per the GUI make the R capital or not in the registry.